awx_collection/plugins/modules/schedule.py

```diff
@@ -214,14 +214,16 @@ def main():
     if inventory:
         inventory_id = module.resolve_name_to_id('inventories', inventory)
     search_fields = {}
+    sched_search_fields = {}
     if organization:
         search_fields['organization'] = module.resolve_name_to_id('organizations', organization)
     unified_job_template_id = None
     if unified_job_template:
         search_fields['name'] = unified_job_template
         unified_job_template_id = module.get_one('unified_job_templates', **{'data': search_fields})['id']
+        sched_search_fields['unified_job_template'] = unified_job_template_id
```

Comment:
Could we either add information in the docs (line 107) and/or add an example which states that the unified_job_template field will be used to search for the schedule if provided. For the use case of deleting a schedule where there are 2 with the same name that might be useful.